test_runner/regress/test_tenant_conf.py

```diff
@@ -403,3 +407,62 @@ def get_metric():
     metric = get_metric()
     assert int(metric.labels["low_threshold_secs"]) == 24 * 60 * 60, "label resets to default"
     assert int(metric.value) == 0, "value resets to default"
+
+
+@pytest.fixture
+def unknown_fields_env(neon_env_builder: NeonEnvBuilder) -> Generator[NeonEnv, None, None]:
+    env = neon_env_builder.init_start()
+    yield env
+    env.pageserver.allowed_errors.extend(
+        [
```

Comment:
do we really need e2e tests that take considerably more time to run versus using unit tests ?

Reply:
What about running these 4 unknown field tests in a single test case? I think it would be significantly faster as we only start the full cluster once in this case.

Reply:
Now unit test is used to check the error, resolving